src/comments.js

```diff
@@ -272,13 +272,18 @@ function printDanglingComments(path, print, options) {
   const text = options.originalText;
 
   const parts = [];
-  path.each(commentPath => {
```

Comment:
I wonder why it indented this one. Maybe it got messed up because of the memberexpression?